discretisedfield/region.py

```diff
@@ -184,43 +358,43 @@ def edges(self):
         (5, 15, 20)
 
         """
-        return dfu.array2tuple(np.abs(np.subtract(self.p1, self.p2)))
+        return np.abs(np.subtract(self.pmin, self.pmax))
```

Comment:
@samjrholt Would the following work (I think it should):
```suggestion
        return self.pmax - self.pmin
```